src/react_native/status.cljs

```diff
@@ -0,0 +1,47 @@
+(ns react-native.status
```

Comment:
I think it's fine to leave this here for now. 
but imo `react-native.status` is a namespace that does not make clear of this functionality. Perhaps we can have this file in some where more descriptive to the purpose of this codes functionality.
maybe status-go.status ? 🤔

Reply:
yeah better to move it outside react-native because it is not a kinda regular rn library, so probably have something like a `backend` root namespace? `status-go` we could use as well but it might change in the future

Reply:
Reverted native.status related change
Created a separate issue for this https://github.com/status-im/status-mobile/issues/14581
lots unknown work if try to migrate all functions